content/work/coachhub.rst

```diff
@@ -2,26 +2,27 @@ public: yes
 location: work
 grid_template: true
 image:
-  - src: '2016/coachhub/feature.jpg'
+  - src: 'projects/coachhub.jpg'
 client:
   - name: 'ORCAS'
     url: 'http://orcasinc.com'
 project:
   - name: 'CoachHub'
-    tagline: 'Integrated Health Coaching for ORCAS'
+    tagline: 'Integrated Health Coaching'
     url: 'http://www.orcasinc.com/products/coachhub/'
+    case_study: true
```

Comment:
Good point. We can either remove this link for now, or merge the coachhub branch first?